scripts/ci/create-docker-image.sh

```diff
@@ -14,75 +14,87 @@ set -e
 TAG=${REF##*/}
 VERSION=${TAG#v}
 
-[ -n "$VERSION" ] && DOCKER_TAG=$VERSION || DOCKER_TAG=latest
+: ${DOCKER_TAG:=$VERSION}
```

Comment:
what about if VERSION is empty previously `latest` was used ?

Reply:
`REF` is set to "latest" if not set ... as a result `TAG` and `VERSION` are set to "latest"

Reply:
I could be useful to set it if we want to call it by hand

Reply:
ok reverted